uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -763,19 +762,24 @@ sub in_range
     return 0 unless $range_string;
     return 0 unless defined($value);
 
-    my @ranges = split(/,/, $range_string);
+    ## mri_protocol table is being normalized in steps. 
+    ## Until this is completed in full and/or the database 
+    ## prevents such a setting, do not allow projects
+    ## to put comma-separated values in the table columns.
+    ## If they do, LOG the specific scan type as a violated scan  
```

Comment:
@cmadjar @nicolasbrossard 

the comment above summarizes how the comma separated values are being handled. I can basically return 0 or 1 in this function, so I return a 0 as if it is a violated scan (so projects can act on this).
The pull request in Loris #3528 will prevent such a setting as the column type will no longer allow strings